<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HateSpeechAnalayzer</class>
 <widget class="QMainWindow" name="HateSpeechAnalayzer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>717</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hate Speech Analayzer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>580</y>
      <width>761</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>Analyz</string>
    </property>
   </widget>
   <widget class="QLabel" name="Title">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>761</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Hate Speech Analayzer</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter path to your video</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>380</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="textLable">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>340</y>
      <width>361</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>360</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Key word from Private Lexicon</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>390</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>The number of words detected from the private lexicon</string>
    </property>
   </widget>
   <widget class="QLabel" name="lexiconLable">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>360</y>
      <width>391</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="hateWordsCountLable">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>390</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="privateLexiconListView">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>220</y>
      <width>191</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addToLexiconBt">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>220</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeFromLexiconBt">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>250</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="lexiconText">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>180</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="detectionLable">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>450</y>
      <width>761</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>621</width>
      <height>441</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>5</number>
    </property>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>421</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>180</y>
      <width>101</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string>Private lexicon</string>
    </property>
   </widget>
   <zorder>frame</zorder>
   <zorder>pushButton</zorder>
   <zorder>Title</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>textLable</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>lexiconLable</zorder>
   <zorder>hateWordsCountLable</zorder>
   <zorder>privateLexiconListView</zorder>
   <zorder>addToLexiconBt</zorder>
   <zorder>removeFromLexiconBt</zorder>
   <zorder>lexiconText</zorder>
   <zorder>detectionLable</zorder>
   <zorder>label_7</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
